source/install/install-ubuntu-1604-mysql.rst

```diff
@@ -0,0 +1,47 @@
+.. _install-ubuntu-1604-mysql:
```

Comment:
There's a couple of differences. The default MySQL version for 14.04 is 5.5, but 5.6+ is needed, so that requires a different apt-get instruction. The default version on 16.04 is 5.7. Also, 14.04 uses the Upstart daemon, and 16.04 uses systemd.